src/controllers/fcm.controller/fcmController.js

```diff
@@ -0,0 +1,25 @@
+const catchAsync = require('../../utils/catchAsync');
+const { getUserId, UpdateUserFCMToken } = require('./queries/queries');
+const { query: Hasura } = require('../../utils/hasura');
+
+const sendNotification = catchAsync(async (req, res) => {
+  const { fcm_token, cognito_sub } = req.body;
+
+  // Find user id
+  const response = await Hasura(getUserId, {
+    cognito_sub: { _eq: cognito_sub },
+  });
+
+  console.log(response);
+
+  await Hasura(UpdateUserFCMToken, {
+    userId: response.result.data.user[0].id,
```

Comment:
You dont need userId to save the token, you can directly save the token using the cognito_sub. This will save us one db call. Here is the mutation you can use: ```mutation UpdateUserFCMToken($cognito_sub: String!, $fcmToken: String!) {
  update_user(where: {cognito_sub: {_eq: $cognito_sub}}, _set: {fcm_token: $fcmToken}) {
    affected_rows
  }
}```